src/lib/pm_runtime.c

```diff
@@ -21,8 +21,6 @@ static struct pm_runtime_data *prd;
 
 void pm_runtime_init(void)
 {
-	trace_pm("pm_runtime_init()");
-
```

Comment:
Maybe verbose trace? Or in case of debugging future PM-related issues such prints would be added by whoever is doing the debugging?

Reply:
@paulstelian97 Initialization is somewhat logged by trace_point, so there is no need for additional log.

Reply:
Guess I have to implement trace_point at some point (maybe reuse the same memory location for panics as well, as I don't have those implemented on i.MX either). Fair enough then.